database/engine/metadata_log/metadatalog.c

```diff
@@ -285,6 +285,7 @@ void metalog_worker(void* arg)
     struct metalog_cmd cmd;
     unsigned cmd_batch_size;
 
+    sanity_check();
```

Comment:
This is not just fixing a compiler warning, this is adding new code into the code path that was not there previously. Has it been properly tested?

Reply:
This code is not doing anything at runtime. Just running netdata should be enough to test its correctness.

Reply:
Oh, it's that weird macro... sure.